chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/SurveillanceCsvPresenter.java

```diff
@@ -51,13 +50,9 @@ public SurveillanceCsvPresenter(final Properties props) {
      * @param cpList list of Certified Products
      */
     public void presentAsFile(final File file, final List<CertifiedProductSearchDetails> cpList) {
-        FileWriter writer = null;
-        CSVPrinter csvPrinter = null;
-        try {
-            writer = new FileWriter(file);
-            csvPrinter = new CSVPrinter(writer, CSVFormat.EXCEL);
+        try (FileWriter writer = new FileWriter(file);
+                CSVPrinter csvPrinter = new CSVPrinter(writer, CSVFormat.EXCEL);) {
```

Comment:
About half of your multi-resource trys have a semi-colon after the last resource (like this one) and half don't. I'm okay with either way, but we need to pick one and stick with it. I'd prefer the version without the trailing semi-colon, and that's the way I did my trys, but could be persuaded to go the other way with a good reason